src/widgets/img/lv_img.c

```diff
@@ -186,7 +186,13 @@ void lv_img_set_offset_y(lv_obj_t * obj, lv_coord_t y)
 
 void lv_img_set_angle(lv_obj_t * obj, int16_t angle)
 {
-    if(angle < 0 || angle >= 3600) angle = angle % 3600;
+    /* If angle=-3600, then angle=0; if angle=-3601, then angle=3599*/
```

Comment:
I started to use on other places:
```c
while(angle >= 3600) angle -= 3600;
while(angle < 0) angle += 3600;
```

What do you think about it? It seems cleaner to me.

Reply:
Yes, it's much clear. I can accept both. Would you like me to change it to this way?

Reply:
If you agree that it's cleaner, let's use the `while` way.

Reply:
Using while introduces additional loop overhead.

Reply:
Yes, there is overhead. Considering this method is already used eleswhere, we could settle on the loop method?

Reply:
It's not a performance critical part so I value simplicity more here. (i.e. with the `while` loops)